zmon_cli/__main__.py

```diff
@@ -1,7 +0,0 @@
-from zmon_cli.main import main
```

Comment:
why do you remove `__main__.py` ? this was convenience to run `python3 -m zmon_cli`